app/webpacker/controllers/feedback_controller.js

```diff
@@ -0,0 +1,69 @@
+const countKey = 'feedbackPageCount';
+const feedbackDismissedKey = 'feedbackDismissed';
+
+import { Controller } from "stimulus" ;
+
+export default class extends Controller {
+  connect() {
+    if (!this.feedbackDismissed) {
+      this.incrementFeedbackPageCounter();
+
+      if (this.pageViewCount >= 3) {
+        this.show();
+      }
+    }
+  }
+
+  show() {
+    this.element.style.display = 'flex';
+  }
+
+  hide() {
+    this.element.style.display = 'none';
+  }
+
+  dismiss() {
+    this.dismissFeedback();
+    this.hide();
+  }
+
+  // we'll probably want to record the fact that
+  // someone's followed the feedback link
+  giveFeedback() {
+    this.dismiss();
+  }
+
+  incrementFeedbackPageCounter() {
+    const currentValue = this.pageViewCount;
```

Comment:
I think it would be clearer returning `0` from `this.pageViewCount` here instead of dealing with null, then you can drop the conditional and the `pageViewCount` getter becomes:

```
let count = window.localStorage.getItem(countKey) || 0;
return parseInt(count);
```

Don't feel that strongly about it though.